R/cnd-signal.R

```diff
@@ -62,7 +62,7 @@ cnd_signal <- function(cnd, ...) {
         frame <- caller_env()
       }
       if (is_null(cnd$trace)) {
-        info <- abort_context(frame, cnd$parent)
+        info <- abort_context(frame, !is_null(cnd$parent))
```

Comment:
```r
rethrowing <- !is_null(cnd$parent)
info <- abort_context(frame, rethrowing)
```

I would personally find giving this a name a little clearer

Reply:
Like `stack_info`?

Reply:
No I'm literally talking about just giving this intermediate variable a name, rather than just inlining `!is_null(cnd$parent)`. It just makes it a little easier to read the code, otherwise I have to go back to the definition of `abort_context()` to see what the second argument is. This is a very minor comment

Reply:
oh this is in `cnd_signal()`, got it.

https://github.com/r-lib/rlang/commit/321840b776c58008f32d0ffe3e7121117ec7d3ac